src/cursor/cur_backup_incr.c

```diff
@@ -41,7 +42,17 @@ __curbackup_incr_blkmod(WT_SESSION_IMPL *session, WT_BTREE *btree, WT_CURSOR_BAC
     WT_ASSERT(session, cb->incr_src != NULL);
 
     WT_RET(__wt_metadata_search(session, btree->dhandle->name, &config));
+    /*
+     * Check if this is a file with no checkpointed content.
+     */
```

Comment:
Single sentence comments generally go on a single line (there are some exceptions, of course).

Reply:
Fixed.